### DISCUSSION TITLE
Feature Request - Add Folders for Reports

### DISCUSSION THREAD

**Discussion by NickAdolf** [grafana/grafana#61431] [Dashboards, Panels & Visualizations]
Much like dashboards and alerts we have a use case for the same folder structure for Reports so that we can associate 'write' permissions for a given team to a given folder.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82927. Feel free to continue the discussion around this there. Thank you!
